docs/admin/authorization.md

```diff
@@ -72,13 +73,15 @@ to assist in policy management.
 
 ### Policy File Format
 
-For mode `ABAC`, also specify `--authorization_policy_file=SOME_FILENAME`.
+For mode `ABAC`, also specify `--authorization-policy-file=SOME_FILENAME`.
 
 The file format is [one JSON object per line](http://jsonlines.org/).  There should be no enclosing list or map, just
 one map per line.
 
 Each line is a "policy object".  A policy object is a map with the following properties:
-  - `user`, type string; the user-string from `--token_auth_file`
+  - `user`, type string; the user-string from `--token-auth-file`
+  - `group`, type string; it can be empty if the authenticated user is not in any groups,
```

Comment:
that's not quite right... 
* if you specify `user`, it must match the username of the authenticated user
* if you specify `group`, it must match one of the groups of the authenticated user